### PR TITLE
feat(options): add support for domainkey challenges

### PR DESCRIPTION
**Pull request by trieloff**
Pairs well with https://github.com/adobe/franklin-domainkey-provider/pull/35

### CODE REVIEW

File: src/options-pipe.js [hunk 1]
```diff
@@ -10,12 +10,63 @@
...
+  const { domainkey } = state.config;
+  // get slack channel from config
+  const { slack } = state.config;
+  let hash;
+  if (typeof domainkey === 'string') {
+    hash = hashMe(originalHost, domainkey);
+  } else if (Array.isArray(domainkey)) {
+    hash = domainkey.map((dk) => hashMe(originalHost, dk)).join(' ');
+  } else if (typeof slack === 'string') {
+    hash = hashMe(originalHost, slack);
```
Comment (tripodsan) [inline src/options-pipe.js:63]: what if there are several slack channels configured?

File: src/options-pipe.js [hunk 2]
```diff
@@ -10,12 +10,63 @@
...
+  }
+  // get domainkey from config
+  const { domainkey } = state.config;
+  // get slack channel from config
+  const { slack } = state.config;
+  let hash;
+  if (typeof domainkey === 'string') {
+    hash = hashMe(originalHost, domainkey);
+  } else if (Array.isArray(domainkey)) {
+    hash = domainkey.map((dk) => hashMe(originalHost, dk)).join(' ');
```
Comment (tripodsan) [inline src/options-pipe.js:61]: why not add the support for arrays in `hashMe` ?

**Review by tripodsan — CHANGES_REQUESTED** [review]
configured slack channel can also be an array

File: src/options-pipe.js (hunk 2 above)
Comment (trieloff) [reply src/options-pipe.js:61]: So that I can copy and paste it from `helix-rum-provider`

File: src/options-pipe.js (hunk 1 above)
Comment (trieloff) [reply src/options-pipe.js:63]: Ok, I'll add support for that.

File: src/options-pipe.js (hunk 2 above)
Comment (tripodsan) [reply src/options-pipe.js:61]: the 4 lines ?

File: src/options-pipe.js (hunk 1 above)
Comment (trieloff) [reply src/options-pipe.js:70]: done

**Pull request merged**